sklearn/feature_selection/rfe.py

```diff
@@ -184,14 +194,26 @@ def _fit(self, X, y, step_score=None):
             estimator.fit(X[:, features], y)
 
             # Get coefs
-            if hasattr(estimator, 'coef_'):
-                coefs = estimator.coef_
+            if callable(self.importance_getter):
+                coefs = self.importance_getter(estimator)
+            elif isinstance(self.importance_getter, attrgetter):
```

Comment:
An attrgetter should already be callable... But the suggestion here was to handle strings other than 'auto', not to have the user provide an attrgetter

Reply:
I misunderstood your input last time. Changed it as a option to give strings and used `attrgetter` internally.